### ISSUE TITLE
output wrong project property

### ISSUE THREAD

**Issue by purplecabbage** [adobe/aio-cli-plugin-app#217]
**Describe the bug**
In `aio app init` we prompt the user to select a project by name, but it should be selected by title

**Comment by sarahxxu** [comment]
We should use project title instead of project name. project name is not shown in Adobe developer console. 

Current behavior
```
$ aio app init hello
Retrieving information from Adobe I/O Console..
? Select Org Adobe ABC
? Select Project 
  FunnyRedBandicoot 
  ElegantYellowMarten 
  QuickPinkMarmot 
❯ LivePeachMockingbird 
  DrunkBlushPython 
  BalancedBlueDormouse 
  MassivePeachImpala 

```

**Comment by purplecabbage** [comment]
Fixed, and released as 1.1.1
https://github.com/adobe/generator-aio-console/commit/df707ddb078f2b1824c43478a84d85ddd1bf2bea
https://github.com/adobe/generator-aio-console/compare/1.1.0...1.1.1
